app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/MapsActivity.java

```diff
@@ -163,7 +221,7 @@ public void onMapReady(GoogleMap googleMap) {
         mMap = googleMap;
         setMapLongClick(mMap);
         setUpMap();
-        Repository.getInstance().setMap(mMap);
+        Repository.ourInstance.setMap(mMap);
```

Comment:
Did you forget to push changes in the `Repository` class or did the `setMap()` function already exist?